README.md

```diff
@@ -21,34 +43,83 @@ npm install bold-ui --save
 yarn add bold-ui --save
 ```
 
-### Font family: IBM Plex Sans
+> IBM Plex Sans Font family
 
 ```html
 <link href="https://fonts.googleapis.com/css?family=IBM+Plex+Sans:400,400i,700,700i" rel="stylesheet" />
 ```
+---
 
-Visit https://bold.bridge.ufsc.br/getting-started for more information.
+### ✅ Requirements
 
-## Usage
+- [Git](https://git-scm.com/)
+- [Node 14+](https://nodejs.org/)
+- [Yarn](https://yarnpkg.com/)
 
-Quick demonstration of the library usage:
+### 🛠 Used technologies and tools
 
-```jsx
-import React from 'react'
-import ReactDOM from 'react-dom'
-import { Button } from 'bold-ui'
+- [Typescript](https://www.typescriptlang.org/) as the main programming language
+- [React](https://reactjs.org/) as main UI library for the default and reference Bold Design System implementation
+- [Storybook](https://storybook.js.org/) for component documentation and development playground
+- [Jest](https://jestjs.io/) for testing
+- [loki](https://loki.js.org/) for visual regression testing based on the storybook stories
+- [Gatsby](https://www.gatsbyjs.org/) to generate the site documentation (see the `site/` folder)
+- [Prettier](https://prettier.io/) for code formatting
+- [Eslint](https://eslint.org/) for code linting
+- [TravisCI](https://travis-ci.org/laboratoriobridge/bold) for continuous integration
+- [Codecov](https://codecov.io/gh/laboratoriobridge/bold) for code coverage analysis
+- [svgr](https://github.com/gregberge/svgr) for generating the icon components
 
-const App = () => {
-  return <Button kind='primary'>Hello World</Button>
-}
+---
 
-ReactDOM.render(<App />, document.querySelector('#root'))
-```
+### 🏁 Quick start
+
+First run `yarn install` on the project root folder to install all the npm dependencies necessary for the project.
+
+Use `yarn build` to build the entire project and generate the `lib` distribution folder. For a better development experience, use `yarn test:watch` to execute jest tests in watch mode and `yarn storybook` to start the storybook dev server.
+
+> #### Running tests
```

Comment:
```suggestion
#### Running tests
```

The result expected here is to be a h4 in markdown, right?

If yes, do the same thing in the next topics.